api_fintech/settings.py

```diff
@@ -37,7 +37,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'calculator'
+    'calculator',
+    'rest_framework',
```

Comment:
it would be nice to use `rest_framework`, is that it is a great and good framework for building api in django, but since our project is so small ... I think we should rather dispute with @AlexandreProenca